File: api/RS.API/RS.Model/Candidate/CandidateAssignedUserModel.cs
using RS.ViewModel.Approval;
using RS.ViewModel.User;
using System;
using System.Collections.Generic;
using System.Text;

namespace RS.ViewModel.Candidate
{
    public class CandidateAssignedUserModel
    {
        public int CandidateAssignedUserId { get; set; }
        public int ApprovalEventId { get; set; }
        public Guid CandidateId { get; set; }

        public ApprovalEventViewModel ApprovalEvent { get; set; }
        public CandidateViewModel Candidate { get; set; }

        public Guid UserId { get; set; }
        public UserViewModel User { get; set; }
    }
}

File: web/src/app/webapi/models/candidate-assigned-user-model.ts
/* tslint:disable */
import { ApprovalEventViewModel } from './approval-event-view-model';
import { CandidateViewModel } from './candidate-view-model';
import { UserViewModel } from './user-view-model';

/**
 */
export class CandidateAssignedUserModel {
    candidateAssignedUserId?: number;
    approvalEventId?: number;
    candidateId?: string;
    approvalEvent?: ApprovalEventViewModel;
    candidate?: CandidateViewModel;
    userId?: string;
    user?: UserViewModel;
}
